3567/0/about_arrays.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+# :reek:all:
```

Comment:
26 `:reek:all:` If you want to disable some reek offenses disable it one-by-one and enable after the method where it appear